src/main/kotlin/org/pkl/lsp/PklTextDocumentService.kt

```diff
@@ -51,9 +51,6 @@ class PklTextDocumentService(project: Project) : Component(project), TextDocumen
 
   override fun didOpen(params: DidOpenTextDocumentParams) {
     val uri = URI(params.textDocument.uri)
-    project.virtualFileManager.get(uri)?.let { file ->
-      file.version = params.textDocument.version.toLong()
-    }
```

Comment:
This logic was causing the language server to initialize multiple versions of the standard library, because we initialize it upon startup with a version of `-1`, but we then set it to `1` when the client opens up a stdlib module in the editor.